extensions/ql-vscode/src/cli.ts

```diff
@@ -123,6 +124,11 @@ export class CodeQLCliServer implements Disposable {
    */
   private static CLI_VERSION_WITH_DECOMPILE_KIND_DIL = new SemVer('2.3.0');
 
+  /**
+   * CLI version where --kind=DIL was introduced
+   */
+  private static CLI_VERSION_WITH_LANGUAGE = new SemVer('2.4.1');
```

Comment:
Starting to reach the point where we might want an enum of `SemVer`s for different features :) (no need to do this now)